server/jetstream_api.go

```diff
@@ -558,6 +568,36 @@ type JSApiMetaServerRemoveResponse struct {
 
 const JSApiMetaServerRemoveResponseType = "io.nats.jetstream.api.v1.meta_server_remove_response"
 
+/*
+// JSApiMetaServerStreamInfoRequest will provide which streams are located on a particular server
+type JSApiMetaServerStreamInfoRequest struct {
+	// Server name of the peer to be evacuated.
+	Server string `json:"peer"`
+}
+
+// JSApiMetaServerStreamInfoResponse is the response to a peer evacuation request in the meta group.
+type JSApiMetaServerStreamInfoResponse struct {
+	ApiResponse
+	Success bool                      `json:"success,omitempty"`
+	Content map[string][]StreamConfig `json:"content,omitempty"`
+}
+
+const JSApiMetaServerStreamInfoType = "io.nats.jetstream.api.v1.meta_server_stream_info"
+*/
+
+// JSApiMetaServerStreamMoveRequest will move a stream on a server to another
+// response to this will come as JSApiStreamUpdateResponse/JSApiStreamUpdateResponseType
+type JSApiMetaServerStreamMoveRequest struct {
+	// Server name of the peer to be evacuated.
+	Server string `json:"peer"`
+	// Account name the stream to move is in
+	Account string `json:"account"`
+	// Name of stream to move
+	Stream string `json:"string"`
+	// Ephemeral placement tags for the move
+	Tags []string `json:"tags,omitempty"`
```

Comment:
What happens if these conflict with current tags? Does the request fail?
How do you see these being used?

Reply:
the tags applied are stream placement tags + tags in request. 
This way you can only move within the constraints originally specified.